db/migrate/20240209174912_redo_money_storage.rb

```diff
@@ -0,0 +1,12 @@
+class RedoMoneyStorage < ActiveRecord::Migration[7.2]
+  def change
+    add_column :accounts, :original_balance, :decimal, precision: 19, scale: 4, default: 0.0
+    add_column :accounts, :original_currency, :string, default: "USD"
+    add_column :accounts, :converted_balance, :decimal, precision: 19, scale: 4, default: 0.0
+    add_column :accounts, :converted_currency, :string, default: "USD"
+
+    remove_column :accounts, :balance_cents
+    remove_column :accounts, :balance_currency
+    remove_column :accounts, :currency
```

Comment:
Guessing we're cool with dropping existing balances at this stage?

Reply:
Yeah. Especially since those `balance_` columns were associated with that Money gem. Not worth the hassle.